README.rst

```diff
@@ -1,104 +1,138 @@
+Installation
+============
 
-The code is written and maintained by `eduNEXT`_ and it is what we use to support our own multi-tenant services. It was initially created as an extension of the `microsites` and `site_configurations` features of the Open edX platform, but it has grown to completely replace them in order to support more robust multi tenancy model.
+#. Add this plugin in your Tutor ``config.yml`` with the ``OPENEDX_EXTRA_PIP_REQUIREMENTS`` setting.
 
-If you are looking for professional development or support with multi tenancy or multi sites in the Open edX platform, you can reach out at sales@edunext.co
+   .. code-block:: yaml
+      
+      OPENEDX_EXTRA_PIP_REQUIREMENTS:
+         - eox-tenant=={{version}}
+         
+#. Save the configuration with ``tutor config save``.
+#. Build an open edx image with ``tutor images build openedx``.
+#. Launch your platform with ``tutor local launch``.
 
-.. _openedx plugin: https://github.com/edx/edx-platform/tree/master/openedx/core/djangoapps/plugins
-.. _edx-platform: https://github.com/edx/edx-platform/
-.. _eduNEXT: https://www.edunext.co
+Usage
+=====
 
-Compatibility Notes
---------------------
+Once your instance is running, you can access the Django admin site and locate the ``EDUNEXT OPENEDX MULTITENANCY`` models.
 
-+------------------+-----------------------+
-| Open edX Release | Version               |
-+==================+=======================+
-| Ironwood         | < 3.0                 |
-+------------------+-----------------------+
-| Juniper          | >= 3.0 < 4.0          |
-+------------------+-----------------------+
-| Koa              | >= 4.0 <= 5.1.3       |
-+------------------+-----------------------+
-| Lilac            | >= 4.0 < 6.2          |
-+------------------+-----------------------+
-| Maple            | >= 6.0                |
-+------------------+-----------------------+
-| Nutmeg           | >= 6.2                |
-+------------------+-----------------------+
-| Olive            | >= 8.0                |
-+------------------+-----------------------+
-| Palm             | >= v10.0.0            |
-+------------------+-----------------------+
-| Quince           | >= v11.0.0            |
-+------------------+-----------------------+
-
-**NOTE**: Since 6.2 version, eox-tenant does not support Django 2.2
-
-The following changes to the plugin settings are necessary. If the release you are looking for is
-not listed, then the accumulation of changes from previous releases is enough.
-
-**Ironwood**
+- **Microsites:** Store the microsite configuration.
+- **Routes:** Configure the URL for a tenant.
+- **Tenant configs:** Store the configuration for each tenant.
+- **Tenant organizations:** Link each organization with one or multiple tenants.
```

Comment:
what's the difference between microsites and tenant configs? Is the microsites model still in use?

Reply:
We use the tenant model but the microsite is still not deprecated